app/javascript/client/modal/Modal.tsx

```diff
@@ -39,6 +39,10 @@ const Modal: FC<ModalProps> = () => {
             clickEffect = () => null;
             component = (<SuccessModal />);
             break;
+        case "successModal":
+            clickEffect = () => closeModal();
```

Comment:
might want to change the click effect to `null` to force the user to read the success message and choose an option based on the message?

Reply:
Cool, okay sounds good ... I can check out how it's working and we can chat about it next week? Sounds like it's basically a judgment call, one that Jim will probably end up making in the end 😄